README.md

```diff
@@ -27,44 +26,48 @@
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 HELLO WORLD <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 HELLO MICROVERSE <a name="about-project"></a>
 
 **Hello World** is my first project at Microverse to demonstrate an ability to submit a project for code review.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
 
 <details>
-  <summary>Client</summary>
+  <summary>HTML5 & CSS3</summary>
   <ul>
     <li><a href="https://reactjs.org/">React.js</a></li>
   </ul>
 </details>
 
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **displays Hello Microverse**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 
 
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
 
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
+ A web browser, google chrome preferably.
+ A code editor (e.g vscode) 
 
 
 ### Setup
```

Comment:
- [ ] **Good job so far 👏 👏**, Kindly as suggested by the previous reviewer, I think it would be better to remove these default steps on how to get started, This one of the most important part in a README as it helps the user to use your project, 🙏 

 ##### For Example:

### Prerequisites

In order to run this project you need:
To clone or fork and run it in a browser

<!--
Example command:

```sh
 gem install rails
```
 -->

### Setup

Clone this repository to your desired folder:




```sh
  cd my-folder
  git clone https://github.com/Paulinelali/hello-microverse-project
```
---
You may try to give the guides to the user to get started 🙏